2a_model/src/models/1_metab_multitask/2b_multitask_do_gpp.yml

```diff
@@ -2,7 +2,7 @@ exp_name: "1b_multitask_do_gpp"
 
 site_set: "well_obs"
```

Comment:
Not related to "tmmx" or "tmmn", but is there a reason we use "well_obs" for `site_set` here, but "med_obs" for the others?